README.md

```diff
@@ -8,7 +8,21 @@ This repository contains the source code of several dapp tools
 hand-crafted and maintained by DappHub, along with various Nix
 expressions which are all bundled up into a Nix overlay.
 
-To install tools from this repository follow these instructions:
+Contents:
+
+- [Dapp](./src/dapp)
+- [Seth](./src/seth)
+- [Hevm](./src/hevm)
+- [Ethsign](./src/hevm)
+
+We use `Nix` to for packaging and managing our tools.
+
+All dependencies and tools can be automaticaly installed by running:
+```sh
+curl dapp.tools/install | sh
+```
+
+For a manual install, please follow the following instructions.
```

Comment:
"follow the following" sounds bad but I don't know how to improve it 😢

Reply:
Maybe "heed" the instructions 😏

Reply:
- "have a look at"
- "check out"
- "peruse"
- "enjoy"
- "scope"

Reply:
“follow the instructions below”